mir-ci/mir_ci/cgroups.py

```diff
@@ -0,0 +1,77 @@
+import os
+from typing import Iterator, Callable, TYPE_CHECKING
+import pathlib
+import logging
+import asyncio
+
+logger = logging.getLogger(__name__)
+
+if TYPE_CHECKING:
+    CreateReturnType = asyncio.Task["Cgroup"]
+else:
+    CreateReturnType = asyncio.Task
+
+class Cgroup:
+    def __init__(self, pid: int, path: pathlib.Path) -> None:
+        self.pid = pid
+        self.path = path
+
+    @staticmethod
+    def create(pid: int) -> CreateReturnType: # type: ignore
```

Comment:
If you ignore the type (please add the URL to why it's ignored), is `CreateReturnType` needed, then?